Source/CsDebugScript.Engine/Engine/IDebuggerEngine.cs

```diff
@@ -284,5 +284,22 @@ public interface IDebuggerEngine
         /// </summary>
         /// <param name="process">Process to Terminate.</param>
         void Terminate(Process process);
+
+        /// <summary>
+        /// Adds new breakpoint to the given process.
+        /// </summary>
+        /// <param name="process">Process.</param>
+        /// <param name="expression">Expression to be evaluated into breakpoint.</param>
+        /// <returns>New breakpoint.</returns>
+        IBreakpoint AddBreakpoint(Process process, string expression);
+
```

Comment:
Can we have single function with default value for action parameter set to null?

Reply:
I am not sure that I understand but take a look on new approach with BreakpointSpec and default action being null which resolves to status  break.